webapp-django/crashstats/crashstats/static/crashstats/css/screen.less

```diff
@@ -430,17 +430,9 @@ html.production {
             }
             th.sortable {
                 padding-right: 17px;
-                background: url(../../img/2.0/sort-none.png) no-repeat right center;
             }
             th.sortable:hover {
                 cursor: pointer;
-                background-image: url(../../img/2.0/sort-none-hover.png);
-            }
-            th.sortable.sort-desc {
-                background-image: url(../../img/2.0/sort-desc.png);
-            }
-            th.sortable.sort-asc {
-                background-image: url(../../img/2.0/sort-asc.png);
```

Comment:
pipeline is much stricter. These files don't exist. The paths don't work but there simply is no `sort-none.png` file anywhere so it's not a matter of the path. 

If you reference images that can't be found `./manage.py collectstatic` fails hard. 

If the files didn't exist before, I'm sure no feature would be affected.